src/grt/src/GlobalRouter.tcl

```diff
@@ -198,15 +198,17 @@ sta::define_cmd_args "global_route" {[-guide_file out_file] \
                                   [-critical_nets_percentage percent] \
                                   [-allow_congestion] \
                                   [-allow_overflow] \
-                                  [-verbose]
+                                  [-verbose] \
+                                  [-start_incremental] \
+                                  [-end_incremental]
```

Comment:
Document these options.  How are they intended to be used?

Reply:
- The first flag `start_incremental` will be used to initialize the GRT listener (`GRouteDbCbk`) and thus be able to obtain the nets modified by detailed placement
- The second flag `end_incremental` will be used to take those modified nets and only reroute them on the last GRT call.

The flow would be:
```
global_route -start_incremental
detailed_placement
global_route -end_incremental
```

Reply:
I added in the README the description of both flags, do I need to add the example flow as well?
Example flow:
```
global_route -start_incremental
detailed_placement
global_route -end_incremental
```